api/src/main/java/com/capitalone/dashboard/service/DashboardServiceImpl.java

```diff
@@ -20,7 +20,8 @@
     private final ComponentRepository componentRepository;
     private final CollectorRepository collectorRepository;
     private final CollectorItemRepository collectorItemRepository;
-    private final PipelineRepository pipelineRepository;
+    @SuppressWarnings("unused")
```

Comment:
I left this on e but added TODO to remove the variable from the contractor param list, not sure if the owner has plans to use this.